docker-compose.yaml

```diff
@@ -1,7 +1,7 @@
 # create a docker image with mysql and a default password of admin
 version: '3'
 
-name: typehero
+#name: typehero
```

Comment:
put this back please <3

Reply:
I'm going to leave this open until I close the PR. On ubuntu this line breaks the docker compose.

Reply:
No shot the above is true. It'd be helpful to share errors messages and we can guide.

pls revert

Reply:
its definitely in the specification as "name": https://github.com/compose-spec/compose-spec/blob/main/schema/compose-spec.json

However, I get the error: 
```shell
ERROR: The Compose file './docker-compose.yaml' is invalid because:
'name' does not match any of the regexes: '^x-'
```

However, that error is a "property match" thing, which is in the specification json linked above. I see "version" is also ignored now, I wonder if name: property is just borked on ubuntu 20.04?

Reply:
I removed version in this commit https://github.com/typehero/typehero/pull/1585#discussion_r1803158858

id update your docker version cause  I can't repo it.

Reply:
Updated docker, my previous install was definitely weird. Still no-bueno, will have to try in vc at some point to fix.

Reply:
ha! figured it out:
```shell
sudo docker-compose up
```
produces the error, however
```shell
sudo docker compose up
```
avoids the error.